standalone-cli/scripts/checksum.mjs

```diff
@@ -0,0 +1,27 @@
+import { createHash } from 'node:crypto'
+import { readFile, writeFile } from 'node:fs/promises'
+import * as path from 'node:path'
+
+const files = [
+  './tailwindcss-linux-arm64',
+  './tailwindcss-linux-armv7',
+  './tailwindcss-linux-x64',
+  './tailwindcss-macos-arm64',
+  './tailwindcss-macos-x64',
+  './tailwindcss-windows-arm64.exe',
+  './tailwindcss-windows-x64.exe',
+]
+
+const __dirname = path.dirname(new URL(import.meta.url).pathname)
```

Comment:
Learned today that this will fail on Windows and you're supposed to use the `fileURLToPath` helper 🙈  https://github.com/tailwindlabs/tailwindcss/pull/14065/files#diff-8cdc2dcef376d2bbc6b8c9029a227c928f2aaa0f460776ba24d9d3eba110aab1

Reply:
Luckily, we don't have to worry about this for this particular case since it will be executed on a linux machine